libs/aws/langchain_aws/agents/utils.py

```diff
@@ -20,29 +20,62 @@
 
 logger = logging.getLogger(__name__)
 
+__version__ = "0.1.0"
```

Comment:
`__version__` has a special meaning in python, which signifies the version of the module loaded. That's not the intent here. Should we use `bedrock_agents_version` here? Might also be useful to add a note to update this value for future updates in Bedrock.